src/components/Header/SideNavbar.js

```diff
@@ -0,0 +1,108 @@
+import React from 'react'
+import styled from 'styled-components'
+import remcalc from 'remcalc'
+import Flex from 'styled-flex-component'
+import is from 'styled-is'
+
+import close from '../../images/close.svg'
+import SideNavAnchor from './SideNavAnchor'
+
+const MobileCloseButton = styled.button`
+  min-width: ${props => remcalc(props.theme.elementSizes.tappableArea)};
+  min-height: ${props => remcalc(props.theme.elementSizes.tappableArea)};
+  background: transparent;
+  color: ${props => props.theme.colors.white};
+  border: none;
+  font-size: ${remcalc(40)};
+  height: ${remcalc(24)};
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-end;
+  margin: ${remcalc(24)};
+
+  &:focus {
+    background: transparent;
+    outline: ${remcalc(4)} solid ${props => props.theme.colors.outline};
+    color: ${props => props.theme.colors.text};
+  }
+`
+const SideNavList = styled.ul`
+  padding-left: ${remcalc(24)};
+`
+
+export const SideNavListItem = styled.li`
+  list-style-type: none;
+
+  &:not(:last-child) {
+    margin-bottom: ${remcalc(18)};
+  }
+
+  &:last-child > a {
+    padding-right: 0;
+  }
+`
+const SideNavModal = styled(Flex).attrs({
```

Comment:
why the `Modal` name?

Reply:
It behaves like a modal, no? :woman_shrugging: 

This `Flex` is the one with the background colour - it's the modal that pops up over the top of the rest of the page.

Reply:
are you talking about this dark menu?

![image](https://user-images.githubusercontent.com/3236388/53506700-74371c00-3aae-11e9-91d0-6a9bb3a99bbe.png)

If so, I guess it should be called `SideNavPanel` or `SideNav`

Reply:
Yeah but the name of the wrapper `div` (SideNavModal/SideNavPanel) + its contents (close button + list of links) is already called `SideNav.

But if "Panel" is more accurate than "Modal" for this type of thing, I can change it to that :+1: